Tests/DangerTests/GitDiffTests.swift

```diff
@@ -52,28 +71,34 @@ final class GitDiffTests: XCTestCase {
                                    FileDiff.Line(text: "", changeType: .added),
                                    FileDiff.Line(text: "", changeType: .added),
                                    FileDiff.Line(text: "protocol SPMCoverageParsing {", changeType: .unchanged),
-                                   FileDiff.Line(text: "    static func coverage(spmCoverageFolder: String, files: [String]) throws -> Report", changeType: .removed),
+                                   FileDiff.Line(text: "    static func coverage(spmCoverageFolder: String, files: [String]) throws -> Report",
```

Comment:
<!--
  0 failure: 
  1 warning:  Line should be 14...
  
  
  DangerID: danger-id-Danger;
  File: Tests/DangerTests/GitDiffTests.swift;
  Line: 74;
-->

- :warning: Line should be 140 characters or less: currently 143 characters (`line_length`)